recipe/meta.yaml

```diff
@@ -1,36 +1,47 @@
 {% set name = "pypng" %}
 {% set version = "0.20220715.0" %}
```

Comment:
Do we need exactly this version or the [latest](https://gitlab.com/drj11/pypng/-/blob/pypng-0.20231004.0/setup.cfg?ref_type=tags) is also fine? See [diff](https://gitlab.com/drj11/pypng/-/compare/pypng-0.20220715.0...pypng-0.20231004.0?from_project_id=37733345&straight=false)

Reply:
Ah, I didn't notice this as I was using pypi.  Since, in this case, the author hasn't deployed the 2023 version to [pypi](https://pypi.org/project/pypng/), I think it will be safer to deploy this version.